test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/exemptionandlosses/ExemptionAndLossesControllerSpec.scala

```diff
@@ -1424,6 +1425,12 @@ class ExemptionAndLossesControllerSpec
           errorArgs: _*
         )
         doc.title() should startWith("Error:")
+
+        val topErrorMsg     = doc.select(s"""a[href="#$key"]""").text()
+        val inlineErrorMsg  = doc.select(s"""span[id="$key-inline-error"]""").text()
+        val topErrorMessage = if (topErrorMsg.nonEmpty) "Error: " + topErrorMsg else topErrorMsg
+
+        topErrorMessage shouldEqual inlineErrorMsg
```

Comment:
could we rename references to `topError` to `errorSummary` so it's clearer. Also, if we're testing a form error, the error summary should always have something in it, so I'm not sure if testing for an empty error summary is appropriate

Reply:
the inline error message wouldn't include the "Error: " prefix would it?

Reply:
When I inspect inline error message its something like this

```
<span class="error-message" id="annualExemptAmount-inline-error">
  <span class="visuallyhidden">Error: </span>
  Enter how much of your Capital Gains Tax Annual Exempt Amount you want to use, like 0 or 12,000
</span>
```

Reply:
ah ok, then - that's ok then. We just need to have a think about whether or not should ever have an error summary that's empty. I don't think we should if this method is to specifically test for form errors